hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/ReadableAccountStore.java

```diff
@@ -47,9 +49,9 @@
  */
 public class ReadableAccountStore implements AccountKeyLookup {
     /** The underlying data storage class that holds the account data. */
-    private final ReadableKVState<Long, MerkleAccount> accountState;
+    protected final ReadableKVState<Long, MerkleAccount> accountState;
```

Comment:
Instead of defining fields as protected it might be better to provide access methods as proctected and keep the fields private.

Reply:
It can be private. I thought of adding `ReadableContractStore extends ReadableAccountStore`. But I did not know how to do with modules. I am not sure if smart contract service module should depend on every other module